pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -2697,6 +2697,104 @@ The azimuth is angle is referenced from north, and is positive clockwise: North
 			tree.VolatilityImmutable,
 		),
 	),
+	"st_frechetdistance": makeBuiltin(
+		defProps(),
+		geometryOverload2(
+			func(ctx *tree.EvalContext, a, b *tree.DGeometry) (tree.Datum, error) {
+				ret, err := geomfn.FrechetDistance(a.Geometry, b.Geometry)
+				if err != nil {
+					return nil, err
+				}
+				if ret == nil {
+					return tree.DNull, nil
+				}
+				return tree.NewDFloat(tree.DFloat(*ret)), nil
+			},
+			types.Float,
+			infoBuilder{
+				info:         `Returns the Frechet distance between the given geometries.`,
+				libraryUsage: usesGEOS,
+			},
+			tree.VolatilityImmutable,
+		),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"geometry_a", types.Geometry},
+				{"geometry_b", types.Geometry},
+				{"densifyFrac", types.Float},
```

Comment:
nit: we usually do snake case for arguments -- can you rename this `densify_frac`?
(remember to regenerate with `make buildshort` afterwards).

Reply:
Ah, my bad - done!